### ISSUE TITLE
Can I have more information about smaller dataset?

### ISSUE THREAD

**Issue by leconize** [hugochan/Eye-Tracker#4]
Hi Hugo. I see you have mentioned the small dataset in the readme. Can you provide me more information about the datasets? I would like to know how you obtain it.

**Comment by hugochan** [comment]
The original dataset comes from the GazeCapture [project](http://gazecapture.csail.mit.edu/). The dataset involves over 1400 subjects and results in more than 2 million face images. Due to the limitation of computation power, a much smaller [dataset](http://hugochan.net/download/eye_tracker_train_and_val.npz) with 48000 training samples and 5000 validation samples was used here. Each sample contains 5 items: face, left eye, right eye, face mask and labels.

**Comment by rofailag18** [comment]
Chan, 
I have three questions:
(1) thank you for providing this nice dataset, but how we would like to separate the validation dataset from the training dataset. It looks like the file you provided is numpy data file. I have to load it using numpy and then save the validation set as a separate data file when using your validation script. 
Also, your validation script validation_script.py requires data file (which I assume this is) and the model file (which you provided as model-23.data-00000-of-00001). 

(2) Once we have the validation data and the model file, I can use these two files as the inputs for validation_script.py, right?

(3) How would your suggest me to test your model with an face image? 

Thank you

**Comment by hugochan** [comment]
@rofailag18 For running the validation script, just run `python validation_script.py pretrained_models/itracker_adv/model-23 -v eye_tracker_train_and_val.npz`.

You might want to split the dataset further into train, validation and test set for fair experiments.

**Comment by FuHongy** [comment]
@hugochan Hi!
I'm wondering that does it need an extra test set when testing the pretrained model?

**Comment by hugochan** [comment]
@FuHongy Yes, it needs. You might want to further split the current training set into training and dev sets and use the current dev set as a test set.
